### PR TITLE
Add Exact TN to docs

### PR DESCRIPTION
**Pull request by multiphaseCFD**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [x] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [x] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**

[sc-72893]

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: doc/lightning_tensor/device.rst [hunk 1]
```diff
@@ -53,13 +60,13 @@ Check out the :doc:`/lightning_tensor/installation` guide for more information.
 
 .. seealso:: `DefaultTensor <https://docs.pennylane.ai/en/latest/code/api/pennylane.devices.default_tensor.DefaultTensor.html>`__ for a CPU only tensor network simulator device.
 
-Note that as ``lightning.tensor`` cannot be cleaned up like other state-vector devices because the data is attached to the graph. It is recommended to create a new ``lightning.tensor`` device per circuit to ensure resources are correctly handled.
+Note that as ``lightning.tensor`` cannot cleaned up the data with `reset()` like other state-vector devices. It is recommended to create a new ``lightning.tensor`` device per circuit to ensure resources are correctly handled.
```
Comment (maliasadi) [inline doc/lightning_tensor/device.rst:63]: Doesn't reset have any docstring?
image
 
```suggestion
Note that as ``lightning.tensor`` cannot cleaned up the data with ``reset()`` like other state-vector devices. It is recommended to create a new ``lightning.tensor`` device per circuit to ensure resources are correctly handled.
```

**Review by maliasadi — CHANGES_REQUESTED** [review]
Thanks @multiphaseCFD! Ping me for the second round :slightly_smiling_face:

File: doc/lightning_tensor/device.rst [hunk 2]
```diff
@@ -41,6 +34,20 @@ Users also have the flexibility to customize these parameters according to their
...
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device("lightning.tensor", wires=100, method="tn")
+
+The ``lightning.tensor`` device dispatches all operations to be performed on a CUDA-capable GPU of generation SM 7.0+ (Volta and later)
+and greater. This device supports both exact and finite shots measurements. Currently, the supported differentiation methods are parameter-shift and finite-diff. Note that the MPS backend of lightning.tensor supports multi-wire gates via Matrix Product Operators (MPO).
+
+The ``lightning.tensor`` device is designed for expectation value calculations. Measurements of ``qml.probs()`` or ``qml.state()`` return dense vectors of dimension :math:`2^{n_\text{qubits}}`, so they should only be used for small systems.
```
[the remark this answers is not in the document]
Comment (multiphaseCFD) [reply doc/lightning_tensor/device.rst:47]: Nice one!

File: doc/lightning_tensor/device.rst [hunk 3]
```diff
@@ -41,7 +33,21 @@ Users also have the flexibility to customize these parameters according to their
...
-The ``lightning.tensor`` device allows users to get quantum circuit gradients using the ``parameter-shift`` method. This can be enabled at the PennyLane ``QNode`` level with:
+Users can also run the ``lightning.tensor`` device in the **Exact Tensor Network** mode by setting the ``method`` argument to ``"tn"``:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device("lightning.tensor", wires=100, method="tn")
+
+The lightning.tensor device dispatches all operations to be performed on a CUDA-capable GPU of generation SM 7.0+ (Volta and later)
+and greater. This device supports both exact and finite shots measurements. Currently, the supported differentiation methods are parameter-shift and finite-diff. Note that the MPS backend of lightning.tensor supports multi-wire gates via Matrix Product Operators (MPO).
```
Comment (maliasadi) [inline doc/lightning_tensor/device.rst:44]:
```suggestion
and greater. This device supports both exact and finite shots measurements. Currently, the supported differentiation methods are parameter-shift and finite-diff. Note that the MPS backend of ``lightning.tensor`` supports multi-wire gates via Matrix Product Operators (MPO).
```

File: doc/lightning_tensor/device.rst [hunk 4]
```diff
@@ -128,9 +134,9 @@ The "lightning.tensor" supports all gate operations supported by PennyLane.
 
 **Supported observables:**
 
-The ``lightning.tensor`` supports all observables supported by the Lightning state-vector simulators, besides ``qml.SparseHamiltonian``, ``qml.Projector`` and limited support to ``qml.Hamiltonian``, ``qml.Prod`` since ``lightning.tensor`` only supports 1-wire Hermitian observables.
+The ``lightning.tensor`` supports all observables supported by the Lightning state-vector simulators, besides :class:`~pennylane.SparseHamiltonian`, :class:`~pennylane.Projector` and limited support to :class:`~pennylane.ops.op_math.Sum`, :class:`~pennylane.ops.op_math.Prod` since ``lightning.tensor`` only supports 1-wire :class:`~pennylane.Hermitian` observables.
 
-Users can not create a ``Hamiltonian`` or ``Prod`` observable from ``Hamiltonian`` observables.
+Users can not create a :class:`~pennylane.ops.op_math.Sum` observable or a :class:`~pennylane.ops.op_math.Prod` observable from :class:`~pennylane.ops.op_math.Sum` observables.
```
Comment (maliasadi) [inline doc/lightning_tensor/device.rst:139]: :+1:

**Review by maliasadi — APPROVED** [review]
LGTM! :star:  We need one more approval from @DSGuala or @AntonNI8 to merge the PR.

File: doc/lightning_tensor/device.rst [hunk 5]
```diff
@@ -1,35 +1,27 @@
...
 =======================
 
-The ``lightning.tensor`` device is a tensor network simulator device. The device is built on top of the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`__ from the NVIDIA cuQuantum SDK, enabling GPU-accelerated simulation of quantum tensor network evolution. This device is designed to simulate large-scale quantum circuits using tensor networks. For small circuits, state-vector simulator plugins may be more suitable.
+The ``lightning.tensor`` device is a tensor network simulator, supporting both the Matrix Product State (MPS) and Exact Tensor Network methods. The device is built on top of the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`__ from the NVIDIA cuQuantum SDK, enabling GPU-accelerated simulation of quantum tensor network evolution. This device is designed to simulate large-scale quantum circuits using tensor networks. For small circuits, state-vector simulator plugins may be more suitable.
 
-A ``lightning.tensor`` device can be loaded simply using:
+A ``lightning.tensor`` device is defaulted by the Matrix Product State (MPS) method and can be loaded simply using:
```
Comment (AntonNI8) [inline doc/lightning_tensor/device.rst:6]:
```suggestion
The ``lightning.tensor`` device defaults to the Matrix Product State (MPS) method, and can be loaded using:
```

File: doc/lightning_tensor/device.rst [hunk 6]
```diff
@@ -1,35 +1,27 @@
...
 
 .. code-block:: python
 
     import pennylane as qml
     dev = qml.device("lightning.tensor", wires=100)
 
-By default, the device represents the quantum state approximated as a Matrix Product State (MPS).
+By default, the device represents the quantum state approximated as a MPS.
```
Comment (AntonNI8) [inline doc/lightning_tensor/device.rst:13]:
```suggestion
By default, the device represents the quantum state approximated as an MPS.
```
Alternatively,
```suggestion
By default, the device approximates the quantum state using an MPS.
```
or,
```suggestion
By default, the device represents the quantum state via an MPS approximation.
```

File: doc/lightning_tensor/device.rst (hunk 4 above)
Comment (AntonNI8) [inline doc/lightning_tensor/device.rst:139]:
```suggestion
Users cannot create a :class:`~pennylane.ops.op_math.Sum` observable or a :class:`~pennylane.ops.op_math.Prod` observable from :class:`~pennylane.ops.op_math.Sum` observables.
```

**Review by AntonNI8 — APPROVED** [review]

**Pull request merged**